modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,32 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.21]]
+== Neo4j 5.21
+
+=== Updated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW FUNCTIONS YIELD *
+SHOW PROCEDURES YIELD *
+----
+a|
+Introduced a `deprecatedBy` column to `SHOW FUNCTIONS`, and `SHOW PROCEDURES`.
+It is not returned by default in either command.
+
+The column is a `STRING` value of the replacement one should use if the function/procedure is deprecated and `null` otherwise.
```

Comment:
```suggestion
The column is a `STRING` value specifying a replacement function/procedure if the used function/procedure is deprecated. Otherwise, it returns `null`.
```